docs/activities/task-forces/tf-plugfest/index.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Plugfest/Testing Task Force
+group: "navigation"
+---
+<div>
+<h2>W3C WoT – Plugfest/Testing Task Force</h2>
+    <p>The WoT Plugfest/Testing task force is responsible for organizing the Web of 
+        Things (WoT) Plugfests and also for organizing general testing procedures, 
+        data collection, and implementation report tooling for WoT deliverables. 
+    </p>
+    <div>
+        <h3>People</h3>
+	<dl>
+        <dt>TF-Lead:</dt><dd>
+            Fady Salama (Siemens AG)
+        </dd>
+    </div>
```

Comment:
closing tag `</dl>`  is missing before `</div>`

Reply:
Done! Please resolve if fixed